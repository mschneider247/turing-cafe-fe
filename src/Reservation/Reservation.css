.reservation {
  width: 100px;
  margin: 1%;
  background-color: white;
  border: 3px solid teal;
  border-radius: 5%;
}

button {
  background-color: teal;
  margin-bottom: 4%;
}