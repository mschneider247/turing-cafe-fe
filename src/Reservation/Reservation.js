import React from 'react';
import './Reservation.css'

const Reservation = () => {

  return (
    <h3>Reservation</h3>
  )
}

export default Reservation;