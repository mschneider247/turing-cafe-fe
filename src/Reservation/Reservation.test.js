import React from 'react';
import { shallow } from 'enzyme';
import Reservation from './Reservation';

describe('Reservation', () => {

  it('Should match the snapshot', () => {
    const wrapper = shallow(<Reservation />)
    expect(wrapper).toMatchSnapshot();
  })

  it('Should trigger removeReservation when button clicked', () => {
    const removeReservation =  jest.fn();
    const wrapper = shallow(<Reservation removeReservation={removeReservation}/>);
    wrapper.find('button').simulate('click');
    expect(removeReservation).toHaveBeenCalled();
  })
})