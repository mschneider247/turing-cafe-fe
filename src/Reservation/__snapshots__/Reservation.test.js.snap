// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reservation Should match the snapshot 1`] = `
<article
  className="reservation"
>
  <p />
  <p />
  <p />
  <p />
  <button
    onClick={[Function]}
  >
    Cancel
  </button>
</article>
`;
